python/xorbits/_mars/dataframe/base/tests/test_base_execution.py

```diff
@@ -542,6 +542,135 @@ def test_apply_with_arrow_dtype_execution(setup):
     pd.testing.assert_series_equal(result, expected)
 
 
+def test_data_frame_applymap_execute(setup):
```

Comment:
You can add a TODO here to support GPU for ``applymap`` op when you have time and gpu machine.